widget_driver/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -350,7 +350,7 @@
 		};
 		249021D4217E4FDB00AE95B9 /* Profile */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = F5286DBEB732EE185778753C /* Pods-Runner.profile.xcconfig */;
+			baseConfigurationReference = 7AFA3C8E1D35360C0083082E /* Release.xcconfig */;
```

Comment:
I could not build locally before changing these settings. That why these changes are there